cocos/platform/android/libcocos2dx-with-controller/build.gradle

```diff
@@ -26,6 +26,6 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: '../java/libs', include: ['*.jar'])
-    compile fileTree(dir: '../ControllerManualAdapter/libs', include: ['*.jar'])
+    implementation fileTree(dir: '../java/libs', include: ['*.jar'])
+    implementation fileTree(dir: '../ControllerManualAdapter/libs', include: ['*.jar'])
```

Comment:
What's the difference between `compile` and `implementation`?
Any reference of this?

Reply:
`compile` is deprecated in favor of `implementation`  - https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#new_configurations

We could use `api` if that is better suited for it

Reply:
Got it. Seems great.